### PR TITLE
CWS Discovery updates

### PR DESCRIPTION
**Pull request by dotproto**
Add information about the recently announced item badges on Chrome Web Store.

[Staging page](https://deploy-preview-2626--developer-chrome-com.netlify.app/docs/webstore/discovery)

### CODE REVIEW

File: site/en/docs/webstore/discovery/index.md [hunk 1]
```diff
@@ -51,5 +51,59 @@ the home page, items chosen for Editors' Picks must follow our general [best
...
 guarantee being featured. Developers cannot pay to be featured as an Editors' Pick.
 
-[best-practices]: https://developer.chrome.com/docs/webstore/best_practices/
-[best-listing]: https://developer.chrome.com/docs/webstore/best_listing/
+## Item badges {: #badges }
+
+Badges on Chrome Web Store are visible to users as they browse the store, and offer an added signal
+of quality and trust on vetted items. Developers who earn these badges may receive higher rankings
+in search and filtering, and might also see their extensions appear in special promotions both on
+and off Chrome Web Store. Learn more about the badges below.
```
Comment (jpmedley) [inline site/en/docs/webstore/discovery/index.md:59]:
```suggestion
and off Chrome Web Store. Learn more about badges below.
```

File: site/en/docs/webstore/discovery/index.md [hunk 2]
```diff
@@ -51,5 +51,59 @@ the home page, items chosen for Editors' Picks must follow our general [best
...
+### Featured badge {: #featured-badge }
+
+{% Img src="image/WlD8wC6g8khYWPJUsQceQkhXSlv1/FL4P4tqG0jrhTdLuQ4Jp.png", width="800", height="400",
+   alt="Screenshot of an extension listing that has a featured badge.", class="screenshot" %}
+
+The Featured badge is assigned to extensions that follow our technical best practices and meet a
+high standard of user experience and design. Chrome team members manually evaluate each extension
+before it receives the badge, paying special attention to the following:
+
+1. Adherence to Chrome Web Store’s [best practices][best-practices] guidelines, including providing
```
Comment (jpmedley) [inline site/en/docs/webstore/discovery/index.md:70]:
```suggestion
1. Adherence to Chrome Web Store’s [best practices][best-practices], including providing
```

File: site/en/docs/webstore/discovery/index.md [hunk 3]
```diff
@@ -51,5 +51,59 @@ the home page, items chosen for Editors' Picks must follow our general [best
...
+   an enjoyable and intuitive experience, using the latest platform APIs and respecting the [privacy
+   of end-users][user-privacy].
```
Comment (jpmedley) [inline site/en/docs/webstore/discovery/index.md:72]:
```suggestion
   of users][user-privacy].
```

File: site/en/docs/webstore/discovery/index.md [hunk 4]
```diff
@@ -51,5 +51,59 @@ the home page, items chosen for Editors' Picks must follow our general [best
...
+
+The Established Publisher badge showcases developers who have verified their identity and demonstrated compliance with the developer program policies. This badge is granted automatically to publishers who meet the following two conditions:
+
+1. The publisher's identity has been [verified][verified].
+2. The publisher has established a consistent positive track record with Google services and
+   compliance with the Developer Program Policy.
+
+Any extensions built by a developer who follows the Chrome Web Store Developer Program Policies and
+does not have any unresolved violations will be considered. For new developers, it will take at
+least a few months of respecting these conditions to qualify. Today, this represents nearly 75% of
```
Comment (jpmedley) [inline site/en/docs/webstore/discovery/index.md:90]: >at least a few months

Is there no way to be more specific?

Comment (jpmedley) [inline site/en/docs/webstore/discovery/index.md:90]:
```suggestion
least a few months of respecting these conditions to qualify. Today, this represents nearly 75 percent of
```

File: site/en/docs/webstore/discovery/index.md [hunk 5]
```diff
@@ -51,5 +51,59 @@ the home page, items chosen for Editors' Picks must follow our general [best
...
+consideration. The [One Stop Support][one-stop-support] page is trialing an option that allows
+developers to nominate extensions for the Featured badge. To successfully nominate your extension to
+receive the Featured badge, the following additional criteria must be met:
+
+1. Your item must be an extension.
+2. You must own the extension you are nominating.
+3. Your extension should have English language support.
+4. Your extension must be published and public.
+5. Your extension must not have any active policy violations.
+6. Your extension’s core features must be accessible without additional credentials or payments.
```
Comment (jpmedley) [inline site/en/docs/webstore/discovery/index.md:103]:
```suggestion
* Your item must be an extension.
* You must own the extension you are nominating.
* Your extension should have English language support.
* Your extension must be published and public.
* Your extension must not have any active policy violations.
* Your extension’s core features must be accessible without additional credentials or payments.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/webstore/discovery/index.md (hunk 4 above)
Comment (dotproto) [reply site/en/docs/webstore/discovery/index.md:90]: Good question. So far, no, but I will follow up on this.

File: site/en/docs/webstore/discovery/index.md [hunk 6]
```diff
@@ -51,5 +51,59 @@ the home page, items chosen for Editors' Picks must follow our general [best
...
+2. A [store listing page][best-listing] that is clear and helpful for users, with quality images and
+   a detailed description.
+
+### Established Publisher badge {: #publisher-badge }
+
+{% Img src="image/WlD8wC6g8khYWPJUsQceQkhXSlv1/vZQXBcGDZaWBWQLPCM95.png", width="800", height="400",
+   alt="Screenshot of an extension listing that has an established publisher badge.",
+   class="screenshot" %}
+
+The Established Publisher badge showcases developers who have verified their identity and demonstrated compliance with the developer program policies. This badge is granted automatically to publishers who meet the following two conditions:
```
Comment (dotproto) [inline site/en/docs/webstore/discovery/index.md:82]:
```suggestion
The Established Publisher badge showcases developers who have verified their identity and
demonstrated compliance with the developer program policies. This badge is granted automatically to
publishers who meet the following two conditions:
```

**Comment by dotproto** [comment]
@jpmedley I'm merging this in prep for tomorrow morning. If you have any outstanding concerns please note them on this PR and I will create a new PR to address them.

**Pull request merged**